components/project-details/ProjectDetailsImage.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Image from 'next/image';
+
+const ProjectDetailsImage: React.FC<{ imageUrl: string; imageAlt: string }> = ({
```

Comment:
Try to be more consistent throughout your app. Define your types in a separate folder as you have done previously in your code.